service/src/main/java/bio/terra/pipelines/service/PipelinesService.java

```diff
@@ -122,7 +122,10 @@ public Pipeline updatePipelineWorkspace(
     }
 
     // ensure that major version of wdlMethodVersion matches the value of the pipeline version.
-    if (pipeline.getVersion().equals(Integer.parseInt(wdlMethodVersion.split("\\.")[0]))) {
+    // split wdlMethodVersion by 'v' and take the last element of the resulting array
+    String splitWdlMethodVersion =
```

Comment:
minor: can we call this "wdlMethodExtractedSemVer" or something? "split" suggested to me that it was the array of strings after the split, not the last element in the array